### DISCUSSION TITLE
Can the GCSToBigQueryOperator accept custom null markers?

### DISCUSSION THREAD

**Discussion by calum-mcg** [apache/airflow#34694] [Q&A] [answered]
In the [latest Google docs](https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-csv#csv-options), it's possible to pass custom null markers when creating a table from a file in GCS, e.g.:

```
    bq load --autodetect \
        --field_delimiter=tab \
        --null_marker="\N" \
        --quote "" \
        --replace --source_format=CSV \
        --skip_leading_rows=1 \
        dataset.table \
        file 
```

Is it possible to do this with `GCSToBigQueryOperator`?

**Comment by calum-mcg** [comment]
Can see that the [corresponding mapping](https://github.com/apache/airflow/blob/f497b72fc021e33a4b8543bb0750bffbb8fe0055/airflow/providers/google/cloud/transfers/gcs_to_bigquery.py#L654) is in the source code, but it doesn't seem to get updated.

**Reply by danielmolnar992** [reply, marked as answer]
Hi! I was stuck with the same and when looked at the source code (same as you linked), I found that this is how you can implement it: 

```
    GCSToBigQueryOperator(
        task_id="my_task",
        src_fmt_configs={"nullMarker": "null"},
        ....
    )
```

**Reply by calum-mcg** [reply]
Thanks for the response!!
